scheduled-cloudfunction/variables.tf

```diff
@@ -1,80 +1,36 @@
 variable "scheduled_function_name" {
   type        = string
-  description = "The string that should be used to create resources associated with the module, pubsub queue, etc"
+  description = "The string that should be used to create resources associated with the module, svc account, pubsub queue, etc"
 
   validation {
-    condition     = length(var.scheduled_function_name) < 32
+    condition     = length(var.scheduled_function_name) >= 6 && length(var.scheduled_function_name) < 30
```

Comment:
Maybe update the description to be >= 6 and < 30

Reply:
good catch, thank you